### PR TITLE
Rename federation-v2 to Kubefed

### PR DESCRIPTION
**Pull request by pmorie**
Rename the `federation-v2` project to `kubefed`; associated kubernetes/org request: https://github.com/kubernetes/org/issues/818

### CODE REVIEW

File: sig-multicluster/README.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ The Chairs of the SIG run operations and processes governing the SIG.
 ## Subprojects
 
 The following subprojects are owned by sig-multicluster:
+- **Kubefed**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/Kubefed/master/OWNERS
```
Comment (nikhita) [inline sig-multicluster/README.md]: This needs to be generated by running `make`. 

The test fails because it should be s/Kubefed/kubefed here.

Comment (pmorie) [reply sig-multicluster/README.md]: Weird, i'm not sure this is the right version that i pushed...

**Comment by pmorie** [comment]
@nikhita ptal

**Pull request merged**